### ISSUE TITLE
Tauon won't open with 'segfault 273550'

### ISSUE THREAD

**Issue by Bl4cKn1gh7** [Taiko2k/TauonMusicBox#235]
I have Tauon installed on Arch Linux, and made sure all dependencies are reinstalled and up to date. I also moved my user files directory to see if that was the issue but still had the same error upon starting.

Terminal output:

> $ python /opt/tauon-music-box/tauon.py
> Tauon Music Box v5.2.1
> Copyright 2015-2020 Taiko2k captain.gxj@gmail.com
> 
> Running from installed location
> User files location: /home/jack/.local/share/TauonMusicBox
> Found XDG-Downloads: /home/jack/Downloads
> Found XDG-Music: /home/jack/Music
> Install directory: /opt/tauon-music-box
> Error loading window state file
> Error loading save file
> Database loaded in 0.09 seconds.
> No translation file available
> Using SDL version: 2.0.10
> Using GStreamer as fallback. Some functions disabled
> [1]    273550 segmentation fault (core dumped)  python /opt/tauon-music-box/tauon.py

**Comment by Taiko2k** [comment]
What desktop environment are you using?

**Comment by Bl4cKn1gh7** [comment]
Disabling GStreamer made it start up again, thanks. So is GStreamer just buggy, or is something incorrectly configured? I was using it because I wasn't able to select pulseaudio as an output with the bass library.

**Comment by Taiko2k** [comment]
Its working for me, but yeah seems to be a more common thing to go wrong. No idea what the issue you're experiencing exactly is. 

If you install the `pulseaudio-alsa` package, that should give you a pulseaudio option with bass.

**Comment by Bl4cKn1gh7** [comment]
so I have pulseaudio-alsa installed, but when I select bass I don't see any outputs listed. 
![image](https://user-images.githubusercontent.com/34180098/73618562-8db5e180-45f6-11ea-9b3b-66cbb8ec6de6.png)

**Comment by Taiko2k** [comment]
Oh sorry, doing that actually disabled both. Now that you have bass selected, just un-comment line 29782.

**Comment by Bl4cKn1gh7** [comment]
Uncommented the line and bass is able to play tracks, but I still can't get any audio since there is no option to output to pulse.
![image](https://user-images.githubusercontent.com/34180098/73886993-ac0b1f80-4838-11ea-9ab4-9bf93653b49b.png)

I've had this issue since before tauon stopped opening, which was the main reason I stopped using bass in the first place.

**Comment by Taiko2k** [comment]
It's not there if you scroll down that list? It should appear as "PulseAudio Sound Server" or something like that. (It's a scrollable panel)

I see you have Jack installed... Perhaps something about your configuration is causing the pulseaudio-alsa' config not to work.

Another option could be to try the Flatpak version of Tauon.

**Comment by Bl4cKn1gh7** [comment]
Jack is the only thing that shows up at the end of the list, Pulse isn't there when I scroll down. Bass was working a few versions ago so maybe its a configuration issue on my end..

**Comment by Taiko2k** [comment]
Since GStreamer is crashing, I imagine there must be at least one thing wrong somwhere.

**Comment by Taiko2k** [comment]
Actually I might be a bit confused. `pulseaudio-alsa` doesn't actually show up as Pulseaudio, rather, have you tried select "Default" in the bass output list?

**Comment by Bl4cKn1gh7** [comment]
I managed to fix it by setting my asound.conf back to the pulseaudio-alsa [config](https://git.archlinux.org/svntogit/packages.git/tree/trunk/asound.conf?h=packages/pulseaudio-alsa). I had configured it like this previously:

`# Use PulseAudio plugin hw
pcm.!default {
   type plug
   slave.pcm hw
}`

It had worked, but it was only coming through my headphones with the default output selected. Now it shows up in pulseaudio's volume manager so I can select the output, so all looks fixed.
